<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>704</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>11111111</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #8db596;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>520</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>750</width>
        <height>520</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>680</width>
           <height>440</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>680</width>
           <height>440</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_3">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>420</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>420</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="yourListLabel">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: italic 20pt &quot;Courier New&quot;;
color: rgb(3, 3, 3)</string>
               </property>
               <property name="text">
                <string>Your List</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QListWidget" name="list">
               <property name="minimumSize">
                <size>
                 <width>320</width>
                 <height>370</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>320</width>
                 <height>370</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: #bedbbb;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>350</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="0">
              <widget class="QPushButton" name="addItemButton">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: italic 20pt &quot;Courier New&quot;;
color: rgb(3, 3, 3);
background-color: #92817a;</string>
               </property>
               <property name="text">
                <string>Add Item</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>10</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QTextEdit" name="newItemInput">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>170</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>170</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: #bedbbb;
border-style: solid;
border-color: #707070;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="addNewItemLabel">
               <property name="minimumSize">
                <size>
                 <width>210</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>210</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: italic 20pt &quot;Courier New&quot;;
color: rgb(3, 3, 3)</string>
               </property>
               <property name="text">
                <string>Add new Items</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="deleteItemButton">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: italic 20pt &quot;Courier New&quot;;
color: rgb(3, 3, 3);
background-color: #92817a;</string>
               </property>
               <property name="text">
                <string>Delete Item</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="title">
         <property name="styleSheet">
          <string notr="true">font: italic 30pt &quot;Zapfino&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>To Do List</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
